core/tests/wdio.conf.js

```diff
@@ -264,7 +264,7 @@ exports.config = {
   // the path portion of your baseUrl. If your `url` parameter starts
   // without a scheme or `/` (like `some/path`), the base url gets
   //  prepended directly.
-  baseUrl: 'http://localhost:9001',
+  baseUrl: 'http://localhost:8181',
```

Comment:
Will this change effect the tests in python setup?

Reply:
no, because I changed the value for the `e2e PORT number` also from where it was taking its value to localhost. I made these changes after consulting it with @U8NWXD